book-examples/leptos/src/default/components/ui.rs

```diff
@@ -2,3 +2,6 @@
 
 #[cfg(feature = "button")]
 pub use shadcn_ui_leptos_button::default as button;
+
+#[cfg(feature = "card")]
+pub use shadcn_ui_leptos_card::default as card;
```

Comment:
We need to ensure the button is also available when compiling card examples:

```suggestion
#[cfg(any(feature = "button", feature = "card"))]
pub use shadcn_ui_leptos_button::default as button;
#[cfg(feature = "card")]
pub use shadcn_ui_leptos_card::default as card;
```